app/static/js/manageDepartments.js

```diff
@@ -2,11 +2,30 @@
 $("#admin").collapse("show");
 
 $(document).ready( function(){
-    x = $('#departmentsTable');
+    x = $('#activeDepartmentsTable');
```

Comment:
It would be better to use descriptive variables instead of x and y

Reply:
I agree, changes have been made